libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -308,6 +312,26 @@ bool IRGeneratorForStatements::visit(FunctionCall const& _functionCall)
 			")\n";
 		break;
 	}
+	case FunctionType::Kind::Assert:
+	case FunctionType::Kind::Require:
+	{
+		solAssert(arguments.size() > 0, "Expected at least one parameter for require/assert");
+
+		for (auto& arg: arguments)
+			arg->accept(*this);
```

Comment:
Is there a reason why this is done in `visit(FunctionCall)` instead of `endVisit(FunctionCall)`? It could avoid calling `accept` explicitly.

Reply:
No specific reason other than I wanted to use the existing switch. I can change it

Reply:
This won't be done in this PR just yet because @chriseth already did this change in another PR

Reply:
Alright, it's the end.. visit now.